dbms/src/Storages/MergeTree/MergeTreeData.h

```diff
@@ -332,6 +332,7 @@ class MergeTreeData : public IStorage
                   const String & full_path_,
                   const ColumnsDescription & columns_,
                   const IndicesDescription & indices_,
+                  const ConstraintsDescription & constraints_,
```

Comment:
Do we need this field?

Reply:
Yes, we have to set constraints in `dbms/src/Storages/MergeTree/MergeTreeData.cpp:125`